docs/extensions.rst

```diff
@@ -146,6 +146,22 @@ ASGI servers that implement this extension will provide
         },
     }
 
+Servers that support the optional ``offset`` and ``count`` parameters for
+range requests may advertise this capability via::
+
+    "scope": {
+        ...
+        "extensions": {
+            "http.response.pathsend": {
+                "ranges": True,
+            },
+        },
+    }
+
+However, capability signaling is optional; applications may attempt to use
+``offset`` and ``count`` parameters regardless of this flag, and servers
+should handle these parameters if provided.
```

Comment:
wasn't sure on the convention here; if a framework doesn't explicitly convey support for `ranges`, should the server still attempt to read the `offset`/`count` and utilize it if it exists? Or do we generally try to avoid that "fuzzy capability matching" in asgi servers/frameworks?